src/test/java/org/folio/tlib/postgres/TenantPgPoolTest.java

```diff
@@ -216,4 +217,22 @@ public void testSSL(TestContext context) throws IOException {
     }));
   }
 
+  @Test
+  public void size() {
+    TenantPgPool pool = TenantPgPool.pool(vertx, "diku");
+    assertThat(pool.size(), is(0));
```

Comment:
Can you also use the pool and check that it increases size() to 1?

Reply:
ok

Reply:
ok